project/settings/base.py

```diff
@@ -27,6 +27,11 @@
 
     'automationcommon',
     'ucamwebauth',
+    'crispy_forms',
```

Comment:
Based on https://django-filter.readthedocs.io/en/master/guide/rest_framework.html?highlight=%22django-crispy-forms%22 this is a bit flacky "Note that this isn’t actively supported, although pull requests for bug fixes are welcome." I'd include a note to bear that in mind as it seems that the project is a bit dead: https://github.com/django-crispy-forms/django-crispy-forms/pulls

Reply:
OK, we can probably remove it.